frontend/app/work_packages/controllers/dialogs/columns.js

```diff
@@ -35,13 +35,16 @@ module.exports = function($scope, $filter, columnsModal, QueryService, WorkPacka
   vm.selectedColumns = [];
   vm.oldSelectedColumns = [];
   vm.availableColumns = [];
+  vm.unusedColumns = [];
```

Comment:
What is the semantic difference between an available column and an unused column? Are both lists equal?